whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdate.java

```diff
@@ -50,7 +48,7 @@ private String getHeader() {
                 " - From-Host: %s\n" +
                 " - Date/Time: %s\n",
                 remoteAddress,
-                DATE_FORMAT.format(dateTimeProvider.getCurrentDateTime()));
+                FormatHelper.dayDateTimeToUtcString(dateTimeProvider.getCurrentDateTime()));
```

Comment:
Can we get rid of ```dateTimeProvider.getCurrentDateTimeUtc()``` ?

Reply:
In theory yes. but that would mean the places where we are using `dateTimeProvider.getCurrentDateTime()` to just get the current time without timezone information would have to updated to something like `dateTimeProvider.getCurrentZonedDateTime().toLocalDateTime()`...this could blow up the diff further.

If you think it is still worth the effort perhaps it can be done as a separate task after the current changes are pushed out of the door?

Reply:
Thanks for checking, no it's OK for now, we can refactor at some other time